.github/workflows/publish-pypi.yml

```diff
@@ -1,3 +1,6 @@
+# PyPI trusted publisher config will need an update if this file is renamed.
+# See: https://docs.pypi.org/trusted-publishers/adding-a-publisher/
+
 name: release
```

Comment:
I needed to give the workflow `id-token: write` permissions but it works well on my fork otherwise!

https://github.com/lgarber-akamai/py-metadata/actions/runs/11261639599/job/31315519864

Reply:
Thanks for catching that! Just fixed it